Makefile.am

```diff
@@ -160,14 +160,23 @@ baseline_filtered_combined.info: leveldb_baseline_filtered.info baseline_filtere
 	$(LCOV) -a leveldb_baseline_filtered.info -a baseline_filtered.info -o $@
 
 test_bitcoin.info: baseline_filtered_combined.info
-	$(MAKE) -C src/ check
+	$(MAKE) -C src/ bitcoin_test_check
```

Comment:
I believe ``check`` is automake's magic rule to run all binaries in the ``TESTS`` variable. Prior to this PR that was only ``test_bitcoin``. This changes ``test_bitcoin.info`` so that it only generates coverage for ``test_bitcoin`` rather than all of ``check``.

I just realized that in a previous attempt (not in this PR) I added ``zcash-gtest`` to ``TESTS``, so this line change was necessary. Later I decided not to add ``zcash-gtest`` to ``TESTS`` because I wasn't sure what all effects that would have for where ``make check`` is used.

So now this is unnecessary. I still think it's an improvement for ``test_bitcoin.info`` because it makes it more specific.